app/services/aapb/batch_ingest/pbcore_xml_ingester_behavior.rb

```diff
@@ -19,9 +19,10 @@ def batch_item_is_digital_instantiation?
         def pbcore_digital_instantiations
           pbcore.instantiations.select { |inst| inst.digital }
         end
-        
+
+        # both physical and unidentified format
         def pbcore_physical_instantiations
-          pbcore.instantiations.select { |inst| inst.physical }
+          pbcore.instantiations.reject { |inst| inst.digital }
```

Comment:
i think you can do the `pbcore.instantiations.reject(&:digital)` fancy syntax here

Reply:
I suppose so... I just mimicked what it did in the digital instantiations method, and what had been done before in this method.